packages/block-editor/src/components/block-breadcrumb/index.js

```diff
@@ -12,6 +12,10 @@ import { chevronRightSmall, Icon } from '@wordpress/icons';
 import BlockTitle from '../block-title';
 import { store as blockEditorStore } from '../../store';
 
+/**
+ * @typedef {import('@wordpress/element').WPElement} WPElement
```

Comment:
Basically all the issues in block-editor were just that we weren't importing types like this.